.buildkite/ftr_security_stateful_configs.yml

```diff
@@ -25,6 +25,10 @@ disabled:
   # Playwright
   - x-pack/test/security_solution_playwright/playwright.config.ts
 
+  # Gen AI Evals run weekly via their own pipeline
+# TODO: Uncomment and remove 'enabled' entry once successful execution is confirmed on CI
+#  - x-pack/test/security_solution_api_integration/test_suites/genai/evaluations/trial_license_complete_tier/configs/ess.config.ts
+
```

Comment:
Note @patrykkopycinski, once done testing on CI, uncomment and remove the below occurrence.